addons/mail/static/src/views/web/fields/emojis_field_common/emojis_field_common.xml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-
```

Comment:
No need to remove this, it may lead to unnecessary conflicts during forward ports